src/components/ReadMore/index.js

```diff
@@ -34,34 +26,31 @@ const ReadMore = (props) => {
     handleLayoutWidth,
     limitNumberLine
   } = useReadmore({
-    numberLine: props.numberLine
+    numberLine: 1
   });
 
   React.useEffect(() => {
     setIsFinishSetLayout(false);
-  }, [layoutWidth, props.text]);
+  }, [layoutWidth, text]);
+
   return (
-    <View onLayout={handleLayoutWidth} style={props.containerStyle}>
+    <View style={{marginLeft: 8, marginTop: 2}} onLayout={handleLayoutWidth}>
       {isFinishSetLayout ? (
-        <TouchableOpacity testID="finishLayout" onPress={props.onPress}>
-          <Text style={[{color: COLORS.gray400}, props.textStyle]}>
+        <TouchableOpacity testID="finishLayout" onPress={onPress}>
+          <Text style={styles.text}>
             {textShown}
-            {''}
             {limitNumberLine < realNumberLine ? (
               <Text testID="moreText" style={styles.moreText}>
                 More...
               </Text>
-            ) : null}{' '}
+            ) : null}
           </Text>
         </TouchableOpacity>
       ) : null}
       {!isFinishSetLayout ? (
-        <TouchableOpacity testID="finishLayout" onPress={props.onPress}>
-          <Text
-            style={[{color: COLORS.gray400}, props.textStyle]}
-            testID="notFinishLayout"
-            onTextLayout={handleLayoutText}>
-            {props.text}{' '}
+        <TouchableOpacity testID="finishLayout" onPress={onPress}>
+          <Text style={styles.text} testID="notFinishLayout" onTextLayout={handleLayoutText}>
+            {text}{' '}
           </Text>
         </TouchableOpacity>
       ) : null}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [18-34]

The `numberLine` prop has been hardcoded to 1 in the `useReadmore` hook. This reduces the flexibility of the `ReadMore` component because it can now only display one line of text before showing the "More..." link. If there's no specific reason for this change, I would suggest reverting it to use a prop so that the number of lines can be customized.

```diff
- numberLine: 1
+ numberLine: props.numberLine
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->